AutowareAuto/src/control/trajectory_follower_nodes/src/lateral_controller_node.cpp

```diff
@@ -163,6 +163,12 @@ LateralController::LateralController(const rclcpp::NodeOptions & node_options)
   // TODO(Frederik.Beaujean) ctor is too long, should factor out parameter declarations
   declareMPCparameters();
 
+  rcutils_ret_t ret = rcutils_logging_set_logger_level("trajectory_follower", RCUTILS_LOG_SEVERITY_DEBUG);
+  if (ret != RCUTILS_RET_OK) {
+    RCLCPP_ERROR(get_logger(), "Could not set log level for traj follower lib");
+
+  }
```

Comment:
we should remove this debugging because it is forcing "trajectory_follower" logger to be DEBUG.